libraries/testing/tester.cpp

```diff
@@ -246,8 +246,7 @@ namespace eosio { namespace testing {
       open( make_protocol_feature_set(), expected_chain_id );
    }
 
-   template <typename Lambda>
-   void base_tester::open( protocol_feature_set&& pfs, fc::optional<chain_id_type> expected_chain_id, Lambda lambda ) {
+   void base_tester::open( protocol_feature_set&& pfs, fc::optional<chain_id_type> expected_chain_id, const std::function<void()>& lambda ) {
```

Comment:
The `lambda()` should be changed to `if( lambda ) lambda();` or use an `fc::optional_delegate` instead of `std::function`